CMakeLists.txt

```diff
@@ -149,19 +149,28 @@ endfunction()
 add_subdirectory(book)
 add_subdirectory(examples)
 
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Core Gui Test REQUIRED)
```

Comment:
Can we have a comment above this block describing what it is doing ? It looks like it is
- Retrieving the install location of Qt
- Passing the qmake location to cargo later (but only for the tests?)
- Settings the target directory to be the `root folder/target`, this is then for the cxx-qt-lib cargo target folder for the tests?

And i assume this is just to help the tests and not relevant for real applications.